src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -5099,13 +5117,22 @@ public void testCreateServiceWithSecretHostnameHostsAndHealthcheck() throws Exce
             .build();
 
     final String[] commandLine = {"ping", "-c4", "localhost"};
+    final long interval = dockerApiVersionLessThan("1.30") ? 30L : 30000000L;
```

Comment:
I think they did, yes. Or maybe they were lenient about the units before, and started being more strict. I think these were always intended to be in nanoseconds (at least [our javadoc](https://github.com/spotify/docker-client/blob/master/src/main/java/com/spotify/docker/client/messages/ContainerConfig.java#L358) says they are in nanoseconds).